src/Equinor.ProCoSys.IPO.Infrastructure/EntityConfigurations/HistoryConfiguration.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Linq;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.HistoryAggregate;
+using Equinor.ProCoSys.IPO.Infrastructure.EntityConfigurations.Extensions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.EntityConfigurations
+{
+    internal class HistoryConfiguration : IEntityTypeConfiguration<History>
+    {
+        public void Configure(EntityTypeBuilder<History> builder)
+        {
+            builder.ConfigurePlant();
+            builder.ConfigureCreationAudit();
+            builder.ConfigureConcurrencyToken();
+
+            builder.Property(x => x.Description)
+                .HasMaxLength(History.DescriptionLengthMax)
+                .IsRequired();
+
+            builder.Property(f => f.EventType)
+                .HasConversion<string>()
+                .IsRequired();
+
+            builder
+                .HasIndex(p => p.ObjectGuid)
+                .HasDatabaseName("IX_History_ObjectGuid_ASC");
```

Comment:
Any particular reason for custom name?

Reply:
No, just taken from preservation. Should I remove the "HasDatabaseName" ? (or all three lines?)